v2/bigquery-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.google.cloud.teleport.v2.templates;
+
+import com.google.api.gax.rpc.InvalidArgumentException;
+import com.google.api.services.bigquery.model.TableReference;
+import com.google.cloud.bigquery.storage.v1beta1.BigQueryStorageClient;
+import com.google.cloud.bigquery.storage.v1beta1.ReadOptions.TableReadOptions;
+import com.google.cloud.bigquery.storage.v1beta1.Storage.CreateReadSessionRequest;
+import com.google.cloud.bigquery.storage.v1beta1.Storage.ReadSession;
+import com.google.cloud.bigquery.storage.v1beta1.TableReferenceProto;
+import java.io.IOException;
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryHelpers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Util class to contain helper factories for interacting with BigQuery directly.
+ */
+public class BigQueryUtils {
```

Comment:
I would split those classes under utils packet. As this functionality is also used in other classes (BigQueryToParquet.java) , it would be nice to add module with shared util classes.